dashboard.py

```diff
@@ -105,6 +105,13 @@ def main() -> None:
 
     print_html5_header()
 
+    # Print a quick table of contents.
+    links = []
+    for kind in PRList._member_map_.values():
+        (id, title) = getIdTitle(kind)
+        links.append(f"<a href=\"#{id}\">{title}</a>")
```

Comment:
What do you think of
```suggestion
        links.append(f"<a href=\"#{id}\" title=\"{title}\">{id}</a>")
```
to save some horizontal space.

I think that reviewers will very quickly map these `id`s to the correct queues.

Reply:
Good idea about the `title` attribute, and I agree about just showing the ID. I've changed this to the short description, which seems even better to me.